st2common/st2common/content/loader.py

```diff
@@ -204,6 +240,36 @@ def _get_folder(self, pack_dir, content_type):
             return None
         return path
 
+    def _get_runners_from_dir(self, base_dir):
+        result = {}
+        for runner_name in os.listdir(base_dir):
+            runner_dir = os.path.join(base_dir, runner_name)
+            runner_manifest_file = os.path.join(runner_dir, RUNNER_MANIFEST_FILE_NAME)
+
+            if os.path.isdir(runner_dir) and os.path.isfile(runner_manifest_file):
```

Comment:
Probably a good idea to add a DEBUG log statement when we found a valid directory and also when we don't (e.g. missing manifest file).